src/kaleidoscope/plugin/Qukeys.h

```diff
@@ -61,7 +60,9 @@ namespace plugin {
 struct Qukey {
  public:
   Qukey(void) {}
-  Qukey(int8_t layer, byte row, byte col, Key alt_keycode);
+  Qukey(int8_t layer, KeyAddr keyAddr, Key alt_keycode);
+  KS_ROW_COL_FUNC Qukey(int8_t layer, byte row, byte col, Key alt_keycode)
+    : Qukey(layer, KeyAddr(row, col), alt_keycode) {}
 
   int8_t layer;
   uint8_t addr;
```

Comment:
This `addr` member should change from `uint8_t` to `KeyAddr` if this PR is adopted. The `addr` is meant to be a single-value identifier for a particular key, after all, and was only used because there was no pre-existing version of that when Qukeys was first written.

Reply:
No, this PR does not intent to mess with any algorithms. Such plugin interna could be delt with by separate PRs.